src/AWS.Deploy.Orchestration/CdkProjectHandler.cs

```diff
@@ -30,6 +31,8 @@ public class CdkProjectHandler : ICdkProjectHandler
         private readonly IDirectoryManager _directoryManager;
         private readonly IAWSResourceQueryer _awsResourceQueryer;
 
+        private const string TemplateIdentifier = "AWS.Deploy.Orchestration.CDK.CDKBootstrapTemplate.yaml";
```

Comment:
Perhaps just for my understanding as someone newer to this project, but how do we decide whether to declare here vs. in `AWS.Deploy.Constants`?

Reply:
We don't really have a specific rule on what goes into the Constants project, but I personally take it on a case by case basis. Generally the Constants project has consts that are used in multiple places and so it made sense to consolidate them. For this specific use case, `TemplateIdentifier` refers to the location of the bootstrap template which is only used in this class and is specific to the Orchestration assembly. It didn't make a lot of sense for me to strip it out and put it in the Constants project.